responsibleai/tests/rai_insights/test_rai_insights_validations.py

```diff
@@ -398,7 +398,7 @@ def test_mismatch_train_test_features(self):
             str(ucve.value)
 
     def test_dirty_train_test_data(self):
```

Comment:
should we add a test with both missing and non-unique data and validate it fails correctly?  It doesn't look like the np.nan -> 1 test change here is validating this scenario?

Reply:
There are tests for missing data. The dirty data test was modified because it had missing value and was failing for missing value reason. the np.nan -> 1 actually validates the dirty data condition.